### DISCUSSION TITLE
Ordinal response variable

### DISCUSSION THREAD

**Discussion by rifatzahan** [cdriveraus/ctsem#48] [General]
Will this package work for ordinal categorical response variable that were collected from patients, three times a day? For example, depression score.

**Comment by cdriveraus** [comment]
Strictly speaking no, it requires continuous or binary measurements, but if you treat your measure as continuous and estimate nonlinear relationships between the latent process and measurements it should be able to get pretty close...

**Reply by rifatzahan** [reply]
Thanks for the reply. Much appreciated.
